svsim/src/main/scala/Backend.scala

```diff
@@ -44,7 +47,8 @@ object CommonSimulationSettings {
     */
   def default = new CommonSimulationSettings(
     plusArgs = Seq.empty,
-    enableWavesAtTimeZero = false
+    enableWavesAtTimeZero = false,
+    traceFileStem = ""
```

Comment:
This can promote the default to live here, i.e.:

```suggestion
    traceFileStem = "trace"
```